src/App.js

```diff
@@ -1,14 +1,28 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-import './App.css';
+import firebase from './firebase/Firebase';
+import "firebase/auth";
 
 import Messages from './containers/Messages/Messages';
 import ContactsList from './containers/Contacts/ContactsList';
 import Login from './containers/Login/Login';
 import Verify from './containers/Login/Verify';
-import NotFound from './NotFound/NotFound';
+import NotFound from './containers/NotFound/NotFound';
+
+import './App.css';
+import './App.scss';
   
 function App() {
+  useEffect(() => {
+    try {
+      firebase.auth().onAuthStateChanged(function(user) {
+        if (user) window.location.pathname = `messages/${user.uid}`;
```

Comment:
We can use the router to navigate instead of window.location which we reload the application.

Reply:
I will work on that after finish the layout. I think more today and I can finish

Reply:
I will do this in the next branch